### DISCUSSION TITLE
Add JetFormBuilder (from CrocoBlock) extension support

### DISCUSSION THREAD

**Discussion by rvdsteege** [pronamic/wp-pronamic-pay#177] [Ideas]
From customer:

> Ik vroeg mij af of Pronamic inmiddels ook ondersteunt wordt voor de JetFormBuilder van CrocoBlock. CrocoBlock is momenteel onwijs populair vanwege de ongekende mogelijkheden.

The free plugin includes support for PayPal and the `jet-form-builder/gateways/register` action allows other gateways to integrate.

- https://crocoblock.com/plugins/jetformbuilder/

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/1568823841/22377_

**Comment by patrickadam** [comment]
Yes. We use it regularly with GF, but would also like to use it with JetFormBuilder.

I had not seen this topic, so I sent Pronamic a message. I got this response: A few years ago we received the same question from you about JetFormBuilder support. Since then we (reference to this topic) have also started tracking requests for this from other users, but there have been none in the past 3 years.

**Comment by rvdsteege** [comment]
Also requested in https://wordpress.org/support/topic/support-for-jetformbuilder/:

> I really like Pronamic Pay, but also the Gutenberg experience JetFormBuilder offerts. JetFormBuilder only supports Stripe and Paypal out of the box. Would it be possible to use Pronamic Pay as a payment gateway for JetFormBuilder?
